.github/workflows/zeebe-benchmark.yml

```diff
@@ -123,30 +123,36 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref }}
+      - uses: ./.github/actions/setup-zeebe
+        with:
+          secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secret_vault_address: ${{ secrets.VAULT_ADDR }}
+          secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+          go: false
+      - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
+        with:
+          maven-extra-args: -PskipFrontendBuild -Dmaven.test.skip=true -am -pl :camunda-zeebe
+          go: false
       - uses: google-github-actions/auth@v2
         id: auth
         with:
           token_format: 'access_token'
           workload_identity_provider: 'projects/628707732411/locations/global/workloadIdentityPools/zeebe-gh-actions/providers/gha-provider'
           service_account: 'zeebe-gh-actions@zeebe-io.iam.gserviceaccount.com'
-      - name: Setup BuildKit
-        uses: docker/setup-buildx-action@v3
       - name: Login to GCR
         uses: docker/login-action@v3
         with:
           registry: gcr.io
           username: oauth2accesstoken
           password: ${{ steps.auth.outputs.access_token }}
-      - uses: docker/build-push-action@v6
+      - uses: ./.github/actions/build-platform-docker
         with:
-          context: .
-          tags: gcr.io/zeebe-io/zeebe:${{ needs.calculate-image-tag.outputs.image-tag }}
+          repository: 'gcr.io/zeebe-io/zeebe'
+          revision: ${{ inputs.ref }}
           push: true
-          cache-from: type=gha,ignore-error=true
-          cache-to: type=gha,mode=max,ignore-error=true
-          build-args: DIST=build
-          provenance: false
```

Comment:
Why was this option removed? It might not be needed for images that are internal-use only but just fyi it can cause problems on older registries: https://github.com/camunda/camunda/commit/1a63c420e952e9fe09e5f6612b349bf820d0f44f

Reply:
This workflow only pushes to gcr.io/zeebe-io which apparently supports provenance.